### PR TITLE
Change structure of js file to add all of the sources that were part …

### PR DESCRIPTION
**Pull request by claudiapao10**
…of the words.json file.

Important files changed:
* information-display.js - added a structure to create multiple sources.
* words.json -changed the path of the images to a general one so it could show up on the extension.

UI:
image

### CODE REVIEW

**Comment by claudiapao10** [comment]
image
image

**Review by sebagabs — APPROVED** [review]
Style is looking good. I think this will serve as our MVP.
@SamRod33 and @SofiaPSU, please try to review this as soon as possible as we have to submit it today to the SPS Project Showcase. I'll be working on the `README.md`.

File: project/chrome-extension/words.json [hunk 1]
```diff
@@ -1,2 +1,2 @@
...
... small changes in Earth\u0027s temperature can have big effects.","image":"/images/climate-change.png","URL":[{"source":"NASA on Climate Change","link":"https://www.nasa.gov/audience/forstudents/k-4/stories/nasa-knows/what-is-climate-change-k4.html"},{"source":"NASA: Earth\u0027s energy budget","link":"https://climate.nasa.gov/news/3072/direct-observations-confirm-that-humans-are-throwing-earths-energy-budget-off-balance/"}]},"Vaccines":{"description":"There is currently no evidence of an increase in miscarriages as a result of COVID-19 immunization. This claim was made based on United Kingdom data.Health officials in the U.K. said the figures are being taken out of context. The article doesn’t account for the increase in the vaccination rates over time, or the average number of miscarriages that occur naturally.","image":"/images/vaccine.png","URL":[{"source":"CDC: Pregnancy Recommendations","link":"https://www.cdc.gov/coronavirus/2019-ncov/vaccines/recommendations/pregnancy.html"},{"source":"Reuters","link":"https://www.reuters.com/article/factcheck-miscarriage-vaccine/fact-check-there-is-no-evidence-to-suggest-covid-19-vaccines-increase-the-risk-of-miscarriage-idUSL1N2LT21A"}]}}
```
Comment (SamRod33) [inline project/chrome-extension/words.json:1]: The pull request in the json-update branch should have reflected these same changes

**Review by SamRod33 — APPROVED** [review]
Very nice! It is so cool seeing it all come together

**Pull request merged**